applications/asset_tracker_v2/src/main.c

```diff
@@ -236,6 +240,10 @@ static void handle_nrf_modem_lib_init_ret(void)
 		break;
 	}
 
+#if defined(CONFIG_NRF_CLOUD_FOTA)
+	(void)nrf_cloud_modem_fota_pending_job_process(ret);
```

Comment:
I might change "process" in the function name to "finalize" or "update" or "finish", just so it is more obvious at a glance that the function modifies the job state. "Process" can mean that something is being processed and thus modified, but it can also mean other things, and it was just barely vague enough to cause me some confusion on first reading

If you decide to leave the name unchanged, then perhaps instead add a comment explaining that the desired effect of the voided call is to update the job status

Reply:
it does not finalize, finish, or update the job state.

i will add a comment as to why the return value is unused; the function always reboots below.